spinnakerresource/pipeline.go

```diff
@@ -124,8 +124,7 @@ func (p Pipeline) validate(localData map[string]interface{}) error {
 	// template is optional key, but it requires defined schema
```

Comment:
I believe that this comment is not actual anymore :)

Reply:
Why? When template key is present, a defined schema is required